tests/phpunit/integration/Core/Authentication/TokenTest.php

```diff
@@ -100,8 +100,11 @@ public function test_set() {
 		$this->assertFalse( $token->set( array( 'expires_in' => 3600 ) ) );
 
 		// Setting only access token also sets reasonable defaults for created and expires_in.
+		$created_before = time();
 		$this->assertTrue( $token->set( array( 'access_token' => 'a1b2c3d4e5' ) ) );
-		$this->assertEquals( time(), $this->user_options->get( OAuth_Client::OPTION_ACCESS_TOKEN_CREATED ) );
+		$created_at = $this->user_options->get( OAuth_Client::OPTION_ACCESS_TOKEN_CREATED );
+		$this->assertGreaterThanOrEqual( $created_before, $created_at );
+		$this->assertLessThanOrEqual( time(), $created_at );
```

Comment:
This fixes a random failure where the time changed in between the lines causing a failure.
https://github.com/google/site-kit-wp/runs/4895422306?check_suite_focus=true#step:11:37

I verified this works as expected by testing with an added delay.

![image](https://user-images.githubusercontent.com/1621608/150519809-5013c3b6-cbc2-4370-bdc8-bf885641a2c7.png)